CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Changed
 - Small refactor of AWS Signer classes for both sync and async clients ([866](https://github.com/opensearch-project/opensearch-py/pull/866))
 - Small refactor to fix overwriting the module files when generating apis ([874](https://github.com/opensearch-project/opensearch-py/pull/874))
+- Fixed a "type ignore" lint error
```

Comment:
For next time, we usually don't need changelog for anything minor like this, we can add a skip-changelog label to the PR.